Gemfile.lock

```diff
@@ -1,32 +1,41 @@
 PATH
   remote: .
   specs:
-    grape-kaminari (0.4.5)
+    grape-kaminari (0.5.0)
```

Comment:
I consider dropping major Ruby version support a breaking change.

But this is the only breaking change -- otherwise all the accumulated changes are dep bumps.